Standards/scs-xxxx-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -0,0 +1,151 @@
+of all stable as well as all upcoming standards. This document describes the standard we use to technically describe certification levels and
+the mandatory standards therein.
+
+# Motivation
+
+This standard has three main objectives:
+- to provide an overview of the mandatory standards for the different SCS certification levels
+- to make the lifecycle of certification levels traceable
+- to provide a machine-readable document for further processing (e.g. for a compliance tool suite or continuous integration).
+
+## Overview of mandatory SCS standards
+
+Digging through a repository of draft, stable, replaced and rejected standards becomes increasingly challenging with a growing number
+documents and decision records. A central document that lists all mandatory standards to aquire a certain certification level can
+resolve this issue. It provides clarity for providers as well as users of sovereign cloud stacks and helps to understand the value
+proposition of SCS.
+
+## Lifecycle of certification levels
+
+Standards and thereas certifications will evolve over time. To provide transparency and traceability for the lifecycle of SCS certification
+levels, the whole history of our certifications should be recorded. Pre-notification of changes to our certification levels also allows
+users to adapt their environments or deployment automation to the new standards in advance.
+
+## Machine-readability for further processing
+
+By providing a machine-readable tool, we can generate web-friendly overviews of our certification levels as well as create a tool suite
+that checks environments against all described standards.
+
+# SCS Certification YAML
+
+Every certification level is recorded in a dedicated YAML file, e.g. `scs-open.yaml`.
+
+The certification YAML *MUST* contain the following keys:
+
+| Key 	| Type 	| Description 	| Example   |
+|-----	|------	|-------------	|---------- |
+| `name` | String | Full name of this certification level	| _SCS Open_ |
+| `url` | String | Valid URL to the latest raw version of this document | _https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Certification/scs-open.yaml_ |
+
+The certification YAML *MAY* contain the following keys:
+
+## Dependency
+
+Standards that are required by lower certification levels shouldn't be included in higher tier certfications again. We thus need to note
+on which certification this level is depending on.
+
+| Key 	| Type 	| Description 	| Example   |
+|-----	|------	|-------------	|---------- |
+| `depends_on` | Map | Preliminary certification level on which this certification depends | |
+| `depends_on.name` | String | Full name of the depending certification level | _SCS Compatible_ |
+| `depends_on.url` | String | Valid URL to the latest raw version of the depending certification level | _https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Certification/scs-compatible.yaml_ |
```

Comment:
There's nothing wrong with referencing well-known and well-defined external standards IMVHO.

Reply:
As we're building this `yaml` for machines, I'd advocate for the full URL. This makes it way easier to write tools that can easily parse multiple, nested `yaml` files. Otherwise, you'd always have to add case switches and switching from one host to another would imply that *all* tools need to be rewritten for the new base URL.